### PR TITLE
Max number of tickets set to 5.

### PR DESCRIPTION
**Pull request by borjaayerdi**

### CODE REVIEW

**Comment by oiertwo** [comment]
:+1:

**Comment by borjaayerdi** [comment]
- Stipe like default option. Remove bank transfers.
- Invoice pdf fixed.  

```
sudo apt-get install wkhtmltopdf
In assopy/assopy/settings.py you need to put the path of wkhtmltopdf. In my case:
WKHTMLTOPDF_PATH = '/usr/bin/wkhtmltopdf'

In epcon/pycon/settings.py
DEFAULT_URL_PREFIX = 'http://127.0.0.1:8000'
```
- Change invoice image, data, bank account and language (spanish & english)
- Limit the number of tickets
- Insert all the countries into the assopy_countries table. 
- If you get a personal ticket change the form to ask (not required) for your personal ID.

TODO
- PayPal.

**Comment by barrachri** [comment]
Hi Borja, sure about this ?
DEFAULT_URL_PREFIX = 'http://127.0.0.1:8000'

**Comment by borjaayerdi** [comment]
Hi Christian,

This was just to have it working locally; of course we will need to change it later and put the conference domain. ;)

Best regards,

Borja

File: p3/templates/p3/cart.html [hunk 1]
```diff
@@ -97,6 +97,27 @@
                             <th>{% trans "Q.ty" %}</th>
                         </tr>
                     </thead>
+
+                <script type="text/javascript">
+                    $("#ticket_table").change(max_tickets);
+                    function max_tickets() {
+                        //alert("The text has been changed.");
```
Comment (d0ugal) [inline p3/templates/p3/cart.html:104]: Looks like this line needs to be removed.

**Pull request merged**